src/Command.js

```diff
@@ -226,3 +226,24 @@ export default class Command {
     throw new Error("command.execute() needs to be implemented.");
   }
 }
+
+export function commandNameFromClassName(className) {
+  return className.replace("Command", "").toLowerCase();
+}
+
+export function exposeCommands(obj, commands) {
```

Comment:
Yeah, that's probably nicer for now.  I had been anticipating a plugin API that added commands after-the-fact, but we can always change the API here if we need to for that.